pyproject.toml

```diff
@@ -24,14 +24,19 @@ dependencies = [
     "PyYAML>=6.0",
     "numpy>=1.21.2",
     "pandas>=1.3.2",
-    "ase>=3.19.0",
+    "ase>=3.22.1",
     "flatdict>=4.0.1",
-    "hyperspy>=1.7.1",
-    "jupyterlab_h5web>=6.0.1",
+    "hyperspy>=1.7.3",
+    "jupyterlab>=3.5.2",
+    "jupyterlab_h5web>=6.6.1",
     "hdfdict @ git+https://github.com/shabihsherjeel/hdfdict@f48a586",
+    "ifes_apt_tc_data_modeling>=0.0.1",
+    "gitpython>=3.1.30",
     "pytz>=2021.1"
```

Comment:
dev-requirements.txt should be updated once pyproject.toml has been modified. That's there to make sure we all have a similar env: `pip-compile --extra=dev --output-file=dev-requirements.txt pyproject.toml`

Reply:
Did this, dev-requirements.txt is part of this pull rq